scripts/version/create-snippet.js

```diff
@@ -1,20 +1,19 @@
 const crypto = require('crypto');
 const fs = require('fs');
 const path = require('path');
-const { version } = require('../../package');
+const yargs = require('yargs');
```

Comment:
Minor, it seems this `yargs` is not used, why do we need this package?

Reply:
Ah yes, need to clean up. Will fix

Reply:
Fixed :)